tools/download-install.sh

```diff
@@ -6,6 +6,22 @@ set -e
 ## Usage: ./download-install.sh                            --> installs the latest version
 ##        TERRA_CLI_VERSION=0.1.0 ./download-install.sh    --> installs version 0.1.0
 
+readonly REQ_JAVA_VERSION=17
+echo "--  Checking if installed Java version is ${REQ_JAVA_VERSION} or higher"
+if [[ -n "$(which java)" ]];
+then
```

Comment:
Plase have "then" on the same line as the "if".
See go/shell-style#control-flow.

Reply:
done, changes in entire repo